src/Testcontainers.WebDriver/WebDriverBuilder.cs

```diff
@@ -0,0 +1,225 @@
+    /// <returns>A configured instance of <see cref="WebDriverBuilder" />.</returns>
+    public WebDriverBuilder WithConfigurationOptions(IReadOnlyDictionary<string, string> options)
+    {
+        return WithEnvironment("SE_OPTS", string.Join(" ", options.Select(option => string.Join("=", option.Key, option.Value))));
+    }
+
+    /// <summary>
+    /// Sets JVM configurations.
+    /// </summary>
+    /// <remarks>
+    /// https://github.com/SeleniumHQ/docker-selenium#se_java_opts-java-environment-options.
+    /// </remarks>
+    /// <param name="javaOptions">A list of JVM configurations.</param>
+    /// <returns>A configured instance of <see cref="WebDriverBuilder" />.</returns>
+    public WebDriverBuilder WithJavaEnvironmentOptions(IEnumerable<string> javaOptions)
+    {
+        return WithEnvironment("SE_JAVA_OPTS", string.Join(" ", javaOptions));
+    }
+
+    /// <summary>
+    /// Sets the screen resolution.
+    /// </summary>
+    /// <remarks>
+    /// https://github.com/SeleniumHQ/docker-selenium#setting-screen-resolution.
+    /// </remarks>
+    /// <param name="width">The screen width.</param>
+    /// <param name="height">The screen height.</param>
+    /// <param name="depth">The screen depth.</param>
+    /// <param name="dpi">The screen dpi.</param>
+    /// <returns>A configured instance of <see cref="WebDriverBuilder" />.</returns>
+    public WebDriverBuilder SettingScreenResolution(int width = 1020, int height = 1360, int depth = 24, int dpi = 96)
+    {
+        IDictionary<string, string> screenResolution = new Dictionary<string, string>();
+        screenResolution.Add("SE_SCREEN_WIDTH", width.ToString());
+        screenResolution.Add("SE_SCREEN_HEIGHT", height.ToString());
+        screenResolution.Add("SE_SCREEN_DEPTH", depth.ToString());
+        screenResolution.Add("SE_SCREEN_DPI", dpi.ToString());
+        return WithEnvironment(new ReadOnlyDictionary<string, string>(screenResolution));
+    }
+
+    /// <summary>
+    /// Sets the session timeout.
+    /// </summary>
+    /// <remarks>
+    /// https://github.com/SeleniumHQ/docker-selenium#grid-url-and-session-timeout.
+    /// </remarks>
+    /// <param name="sessionTimeout">The session timeout.</param>
+    /// <returns>A configured instance of <see cref="WebDriverBuilder" />.</returns>
+    public WebDriverBuilder SetSessionTimeout(TimeSpan sessionTimeout = default)
+    {
+        return WithEnvironment("SE_NODE_SESSION_TIMEOUT", sessionTimeout.TotalSeconds.ToString(CultureInfo.InvariantCulture));
+    }
+
+    /// <summary>
+    /// Sets the time zone.
+    /// </summary>
+    /// <remarks>
+    /// Time Zone Database name (IANA).
+    /// </remarks>
+    /// <param name="timeZone">The time zone.</param>
+    /// <returns>A configured instance of <see cref="WebDriverBuilder" />.</returns>
+    public WebDriverBuilder SetTimeZone(string timeZone)
+    {
+        return WithEnvironment("TZ", timeZone);
+    }
```

Comment:
```suggestion
```

As we previously discussed [here](https://github.com/testcontainers/testcontainers-dotnet/pull/780#discussion_r1130782659), it does not make much sense to expose individual APIs for environment variables. It is always a question of which ones are worth exposing, who keeps them up-to-date, etc. Since developers can simply add environment variables by calling `WithEnvironment(k, v)`, I propose removing these APIs.

Reply:
O.k 
But maybe declarative about that in docs or some example